### PR TITLE
goneovim: Add version nightly

### LINKED ISSUE #482: [Request] Add Goneovim nightly
### Information

**Name:** Goneovim

**Description:** Neovim GUI written in Golang, using a Golang qt backend

**Homepage:** https://github.com/akiyosi/goneovim

**Download link(s):** https://github.com/akiyosi/goneovim/releases/tag/nightly

### PR DESCRIPTION
**Pull request by shiena**
Closes #482

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/goneovim-nightly.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "bin": "goneovim.exe",
+    "shortcuts": [
+        [
+            "goneovim.exe",
+            "Goneovim-nightly"
+        ]
+    ],
+    "checkver": {
+        "url": "https://api.github.com/repos/akiyosi/goneovim/releases/tags/nightly",
+        "regex": "\"target_commitish\": \"([0-9a-f]{7})[0-9a-f]*\"",
```
Comment (rashil2000) [inline bucket/goneovim-nightly.json:25]: Can you try this?

```suggestion
        "regex": "\\"target_commitish\\": \\"([0-9a-f]{7})[0-9a-f]*\\"",
```

**Comment by rashil2000** [comment]
Since commit hash is an alphanumeric value, how will a version _update_ be detected?

**Comment by rashil2000** [comment]
> Since commit hash is an alphanumeric value, how will a version _update_ be detected?

**Comment by shiena** [comment]
There is no explicit version of goneovim-nightly. Therefore, I decided to use the commit hash as the version. But target_commitish is too long, so I want to concatenate "nightly-" with the first 7 characters.

**Pull request merged**